charts/omnirpc/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" }}
```

Comment:
think we may want to allow users do disable?

Reply:
ok